src/main/requests/data/AuthorizedFacebookPagesData.ts

```diff
@@ -0,0 +1,16 @@
+export interface FacebookPage {
+  id: string
+  name: string,
+  access_token: string
+}
+
+/**
+ * Interface to represent the data regarding the list of Facebook
+ * pages this user has authorized to your app.
+ *
+ * @author Andres Gutierrez <andres99@gmail.com>
+ * @since next.release
```

Comment:
Actually needs to be like this:

```suggestion
 * @since `next.release`
```

Same for other places